src/lib/object-mapper/get-key-value.js

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable */
```

Comment:
Easy... I assume this is the ported code... I'm looking at it assuming you're not done with it.